__Made by Ihor Markevych__

# Exercise - Voters

## Problem

In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple random sample of 100 voters are surveyed from each state.

What is the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state?


(A) 0.04   
(B) 0.05   
(C) 0.24   
(D) 0.71   
(E) 0.76  

## Solution

The correct answer is C. For this analysis, let $P_1 =$ the proportion of Republican voters in the first state, $P_2 =$ the proportion of Republican voters in the second state,$p_1 =$ the proportion of Republican voters in the sample from the first state, and $p_2 =$ the proportion of Republican voters in the sample from the second state.
  
    
Let:
* $P_1 = 0.52$, the proportion of Republicans in state 1
* $P_2 = 0.47$, proportion of Republicans in state 2

In [1]:
P1 <- .52
P2 <- .47
cat('P_1 =', P1)
cat('\nP_2 =', P2)

P_1 = 0.52
P_2 = 0.47

## The solution involves four steps

A- Make sure the sample size is big enough to model differences with a normal population.

$$n_1P_1 = 100*.52 = 52$$
$$n_1(1 - P_1) = 100*.48 = 48$$
$$n_2P_2 = 100*.47 = 47$$
$$n_2(1 - P_2) = 100*.53 = 53$$
     
    Conclussion: Each of the quantities above are greater than 10, 
    the sample size is large enough.

In [2]:
n1 <- 100
n2 <- 100
cat('n_1 * P_1 =', n1 * P1)
cat('\nn_1 * (1 - P_1) =', n1 * (1 - P1))
cat('\nn_2 * P_2 =', n2 * P2)
cat('\nn_2 * (1 - P_2) =', n2 * (1 - P2))

n_1 * P_1 = 52
n_1 * (1 - P_1) = 48
n_2 * P_2 = 47
n_2 * (1 - P_2) = 53

B- Find the mean of the difference in sample proportions

$$E(p_1-p_2) = P_1 - P_2 = 0.52 - 0.47 = 0.05$$

In [3]:
cat('E(p_1-p_2) = P_1 - P_2 =', P1 - P2)

E(p_1-p_2) = P_1 - P_2 = 0.05

C- Find the standard deviation of the difference.

$$\sigma=\sqrt{\frac{P_1(1 - P_1)}{n_1} + \frac{P_2(1 - P_2)}{n_2}}$$

In [4]:
cat('sigma =', sqrt(P1 * (1 - P1) / n1 + P2 * (1 - P2) / n2))

sigma = 0.07061869

D- Find the probability. This problem requires us to find the probability that $p_1$ is less than $p_2$. This is equivalent to finding the probability that $p_1 - p_2$ is less than zero. To find this probability, we need to transform the random variable $(p_1 - p_2)$ into a z-score. That transformation appears below.

$$z_{p_1 - p_2} = \frac{x - \mu_{p_1 - p_2}}{\sigma_d} = \frac{0 - 0.05}{0.07061869} = -0.7082$$

Using any distribution calculator, we find that the probability of a z-score being -0.7082 or less is 0.24.


Therefore, the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state is 0.24.

In [5]:
cat('z_{p1 - p2} =', round(pnorm(−0.7082), 2))

z_{p1 - p2} = 0.24